.gitignore

```diff
@@ -24,3 +24,4 @@ coverage
 
 *~
 bots
+./coverage/junit.xml
```

Comment:
We already have `coverage` in .gitignore, shouldn't that be enough to remove the coverage folder from files to commit? :thinking:

Reply:
It should be enough, I will remove it